charts/rancher-istio/1.4.800/questions.yaml

```diff
@@ -1,3 +0,0 @@
-labels:
-  rancher.istio.v1.4.800: 1.4.8
-rancher_min_version: 2.3.7-rc1
```

Comment:
Why was this deleted? 

Let's keep the min version as is.

Reply:
We aren't ever deploying 1.4.8, 1.4.7 is last version from 1.4.9